Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: puppetserver
-version: 9.3.4
-appVersion: 7.13.0
+version: 9.4.0
+appVersion: 7.17.0-v1.5.0
```

Comment:
`-v1.5.0` should be removed everywhere.

Reply:
not sure about that, because thats the container tag, but if you say so